README.md

```diff
@@ -7,15 +7,15 @@ This project is licensed under the Apache-2.0 License.
 ## Dependencies
 | name                       | version              | link                                              |
 |----------------------------|----------------------|---------------------------------------------------|
-| aws-lc                     | v1.0.2               | https://github.com/awslabs/aws-lc/                |
-| S2N                        | v1.3.11              | https://github.com/aws/s2n-tls.git                |
-| aws-c-common               | v0.6.20              | https://github.com/awslabs/aws-c-common           |
+| aws-lc                     | v1.1.0               | https://github.com/awslabs/aws-lc/                |
```

Comment:
I see this is updated to v1.1.0, but in the Dockerfile looks like we are using v1.0.2. Am I missing something?

Reply:
Thanks. Internal copy paste error.